File: test/fixtures/node-request/get_basic_auth_no_user.js
var request = require('request');

var options = {
    url: 'http://localhost:28139/',
    auth: {
        'user': '',
        'pass': 'some_password'
    }
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);

File: test/fixtures/dart/get_basic_auth_no_user.dart
import 'dart:convert';
import 'package:http/http.dart' as http;

void main() async {
  final uname = '';
  final pword = 'some_password';
  final authn = 'Basic ${base64Encode(utf8.encode('$uname:$pword'))}';

  final url = Uri.parse('http://localhost:28139/');

  final res = await http.get(url, headers: {'Authorization': authn});
  final status = res.statusCode;
  if (status != 200) throw Exception('http.get error: statusCode= $status');

  print(res.body);
}
